v2/components/HarvestModal.tsx

```diff
@@ -33,14 +37,17 @@ interface RewardRowPropWrapper {
 const threshold = BigNumber.from("1000000000000");
 
 const RewardRow: FC<RewardRowPropWrapper> = ({ details }) => {
-  const picklePendingAmount = parseFloat(ethers.utils.formatUnits(details.harvestableAmount, 18));
+  const totalEthRewards = parseFloat(ethers.utils.formatEther(details.claimableEthV2));
+  const totalPickleRewards = parseFloat(ethers.utils.formatEther(details.claimableTokenV2.add(details.claimableV1).add(details.harvestableAmount)));
 
-  if (details.harvestableAmount.lt(threshold)) return null;
+  const totalRewards = details.claimableEthV2.add(details.claimableTokenV2).add(details.claimableV1);
+
+  if (totalRewards.lt(threshold)) return null;
```

Comment:
@UrAvgDeveloper this removes rows allowing you to harvest all the available farm rewards from the modal